packages/lodestar/src/db/api/beacon/repository.ts

```diff
@@ -64,15 +64,10 @@ export abstract class BulkRepository<T> extends Repository<T> {
   }
 
   public async getAllBetween(lowerLimit: number, upperLimit: number): Promise<T[]> {
-    let data = await this.db.search({
-      gt: encodeKey(this.bucket, Buffer.alloc(0)),
-      lt: encodeKey(this.bucket + 1, Buffer.alloc(0)),
+    const data = await this.db.search({
+      gt: encodeKey(this.bucket, Buffer.alloc(lowerLimit)),
```

Comment:
remove buffer alloc from gt and lt, encodeKey accepts number @austinabell 

Just so you don't debug too much why this doesnt work :smile:

Reply:
thanks! Just kept as Buffer since that's what it was previously and I saw this could accept a Buffer